crud-module/index.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var fs = require('fs'),
+  inflections = require('underscore.inflections'),
+  mkdirp = require('mkdirp'),
+  s = require('underscore.string'),
+  yeoman = require('yeoman-generator');
+
+var folderChoices = [{
+  value: 'css',
+  name: 'css',
+  checked: false
+}, {
+  value: 'img',
+  name: 'img',
+  checked: false
+}, {
+  value: 'directives',
+  name: 'directives',
+  checked: false
+}, {
+  value: 'filters',
+  name: 'filters',
+  checked: false
+}];
+
+
+var CrudGenerator = yeoman.generators.Base.extend({
+  askForModule: function () {
+    var modulesFolder = process.cwd() + '/modules/';
+    var done = this.async();
+
+    var prompts = [{type: 'input',
+      name: 'name',
+      message: 'What is the name of the CRUD module?',
+      validate: function( value ) {
+        if (value.length > 0) {
+          return true;
+        } else {
+          return "Please enter a CRUD module name";
+        }
+      }
+    },
+      {
+      type: 'checkbox',
+      name: 'folders',
+      message: 'Which supplemental folders would you like to include in your angular module?',
+      choices: folderChoices
+    },{
+      type: 'confirm',
+      name: 'addMenuItems',
+      default: true,
+      message: 'Would you like to add the CRUD module links to a menu?'
+    }];
+
+    this.prompt(prompts, function (props) {
+      this.name = props.name;
+      this.folders = props.folders;
+      this.addMenuItems = props.addMenuItems;
+
+      this.slugifiedName = s.slugify(this.name);
+
+      this.slugifiedPluralName = inflections.pluralize(this.slugifiedName);
+      this.slugifiedSingularName = inflections.singularize(this.slugifiedName);
+
+      this.camelizedPluralName = s.camelize(this.slugifiedPluralName);
+      this.camelizedSingularName = s.camelize(this.slugifiedSingularName);
+
+      this.classifiedPluralName = s.classify(this.slugifiedPluralName);
+      this.classifiedSingularName = s.classify(this.slugifiedSingularName);
+
+      this.humanizedPluralName = s.humanize(this.slugifiedPluralName);
+      this.humanizedSingularName = s.humanize(this.slugifiedSingularName);
+
+      done();
+    }.bind(this));
+  },
+  askForMenuId: function () {
+    if (this.addMenuItems) {
+      var done = this.async();
+
+      var prompts = [{
+        name: 'menuId',
+        message: 'What is your menu identifier(Leave it empty and press ENTER for the default "topbar" menu)?',
+        default: 'topbar'
+      }];
+
+      this.prompt(prompts, function (props) {
+        this.menuId = props.menuId;
+
+        done();
+      }.bind(this));
+    }
+  },
+  createCrudModule: function () {
+    // Create module folder
+    mkdirp('modules/' + this.slugifiedPluralName);
```

Comment:
yes, Create module folder is from the old one but the menu seems needed if you have another menu other than topbar.